src/components/GetStartedBtn.module.css

```diff
@@ -0,0 +1,14 @@
+.getStarted {
```

Comment:
Bad name for a reusable component. Naming it "GetStarted" you set a hard limit of its usage.

I recommend you to create a button component with different variants such as "size" and "variant".

Reply:
I didn’t  understend
insted GetStarted component make BigBtn component? rename?

Reply:
No. BigButton would be another limitation.

Take a look at [Chakra UI](https://v2.chakra-ui.com/docs/components/button) for example. This is how real developers create components. The key idea for each component is reusability.

Reply:
I still don't understand. It’s clear to me how to do this with Chakra UI, but what if I don’t include a this library? Should I pass the 'size' and 'variant' keys and then use them as styling attributes in the component? How does  it  work  without  Chakra UI?

Reply:
It's up to you how to name props. But it's a common practice to give names like that. With `size` you control the size =) with `variant` you can make it more or less rounded in this project.

Reply:
You can explore other options like [Ant.Design](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_objects#accessing_properties) or [Material UI](https://mui.com/material-ui/react-button/)